wire/msgheaders_test.go

```diff
@@ -244,6 +247,7 @@ func TestHeadersWireErrors(t *testing.T) {
 		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, // SBits
 		0x01, 0xe3, 0x62, 0x99, // Nonce
 		0x00, // TxnCount (0 for headers message)
+		//0xca, 0x55, 0xe7, 0x7e, // XXX should we test StakeVersion?
```

Comment:
Yes.  And it needs to be in the correct spot, which would be before the TxnCount.